### PR TITLE
Feature/pending application reviewapplication style button

### PR DESCRIPTION
**Pull request by arsalan2243**
## Description

style improvement on Review application buttons in "pending application"

#### Video Link

https://www.loom.com/share/9ef8013462f841a3bb2fc7ad54a4353c

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/PendingApplications/PendingApplication.css [hunk 1]
```diff
@@ -10,12 +10,43 @@
   padding-top: 20%;
 }
 .note-button-color {
```
Comment (fletchulence) [inline src/components/pages/PendingApplications/PendingApplication.css:12]: This seems to be the same as line 45... why have 2 different ones? Maybe consider putting them into the same place

File: src/components/pages/PendingApplications/PendingApplication.css [hunk 2]
```diff
@@ -10,12 +10,43 @@
...
+  user-select: none;
+  -webkit-user-select: none;
+  vertical-align: top;
+  white-space: nowrap;
+  width: 100%;
+  z-index: 9;
+  border: 0;
 }
+
+.note-button-color {
```
Comment (fletchulence) [inline src/components/pages/PendingApplications/PendingApplication.css:45]: referring to here in the above comment

**Review by fletchulence — APPROVED** [review]
i mean it works i guess!

**Review by ChristianSmith2394 — APPROVED** [review]
Well done, loving the clean uniformity!

**Review by brettsmith212 — APPROVED** [review]
Looks like a simple styling update on the button. Nothing stood out as needing improvement

**Comment by yuririnnnu** [comment]
It looks good to me. Well done!

**Pull request merged**